README.md

```diff
@@ -638,6 +640,7 @@ Services to securely store your Docker images.
 -   [Gebug](https://github.com/moshebe/gebug) - A tool that makes debugging of Dockerized Go applications super easy by enabling Debugger and Hot-Reload features, seamlessly.
 -   [Kitt](https://github.com/senges/kitt) - A portable and disposable Shell environment, based on Docker and Nix. By [@senges](https://github.com/senges)
 -   [Lando](https://github.com/lando/lando) - Lando is for developers who want to quickly specify and painlessly spin up the services and tools needed to develop their projects. By [Tandem](https://thinktandem.io/)
+-   [Rust Universal Compiler](https://github.com/Peco602/rust-universal-compiler) - Container solution to compile Rust projects for Linux, macOS and Windows. By [Peco602](https://github.com/Peco602)
```

Comment:
LGTM

Please add the link to your Github profile to the bottom of the readme markdown and reference it similar to "[@Google][google]" when merging multiple links as our linting will not pass or at least output an alarm.

Reply:
Done